spec/generators/rails-template.rb

```diff
@@ -129,6 +129,11 @@ class Wizard < ApplicationRecord
       has_many :spell_books
 
       scope :recent, -> { where('created_at > ?', 1.month.ago) }
+      
+      if ['5.2', '6.0'].include?(ENV["RAILS_VERSION"])
```

Comment:
I don't love this solution, but it's the least-worst I could come up with. I tried a few other ways to do this, and they were all pretty hacky and bad.

Reply:
I've replaced this with a hack to get it to only add the lines for 5.2 and 6.0, but it's pretty gross. 🤷‍♂